### PR TITLE
Chore upgrade deps

### PR DESCRIPTION
**Pull request by okonet**
**What kind of change does this PR introduce?**

- [ ] bugfix
- [ ] feature
- [x] refactoring / style
- [x] build / chore
- [ ] documentation

**Did you add tests for your changes?**

- [ ] Yes, my code is well tested
- [x] Not relevant

**If relevant, did you update the documentation?**

- [ ] Yes, I've updated the documentation
- [x] Not relevant

**Summary**

Upgrade all the things and fix linting errors

**Does this PR introduce a breaking change?**

No

**Other information**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -75,40 +73,45 @@
...
+    "@commitlint/cli": "^7.0.0",
+    "@commitlint/config-angular": "^7.0.1",
+    "@commitlint/prompt": "^7.0.0",
+    "@commitlint/prompt-cli": "^7.0.0",
     "babel-cli": "^6.9.0",
-    "babel-core": "^6.9.1",
-    "babel-eslint": "^7.1.1",
-    "babel-jest": "^21.0.0",
+    "babel-core": "^6.26.3",
+    "babel-eslint": "*",
```
Comment (TrySound) [inline package.json]: Please, prefer normal caret version.

File: package.json [hunk 2]
```diff
@@ -75,38 +73,43 @@
...
     "imagemin-pngquant": "^5.0.0",
-    "jest": "^21.0.1",
+    "jest": "^23.5.0",
     "jest-enzyme": "^3.8.2",
-    "lint-staged": "^4.1.0",
-    "markdownlint-cli": "^0.3.1",
-    "prettier": "^1.6.1",
+    "lint-staged": "^7.2.2",
+    "markdownlint-cli": "^0.13.0",
+    "prettier": "*",
```
Comment (TrySound) [inline package.json]: Same here. It's better to know which version you use. `yarn upgrade-interactive --latest` will let you upgrade easier.

File: src/index.js [hunk 3]
```diff
@@ -415,12 +417,17 @@ class Dropzone extends React.Component {
       getDataTransferItems,
       ...divProps
     } = props
+    /* eslint-enable no-unused-vars */
 
+    /* eslint-disable jsx-a11y/no-static-element-interactions */
+    /* eslint-disable jsx-a11y/click-events-have-key-events */
```
Comment (TrySound) [inline src/index.js]: Maybe it's easier to disable these rules at all?

File: .eslintrc [hunk 4]
```diff
@@ -1,31 +1,18 @@
...
-  "extends": [
-    "okonet",
-    "prettier"
-  ],
-  "plugins": [
-    "prettier"
-  ],
+  "extends": ["okonet/node"],
+  "parserOptions": {
+    "sourceType": "module"
```
Comment (okonet) [inline .eslintrc]: The only reason is the rollup config. @TrySound is it worth using imports?

File: src/index.js (hunk 3 above)
Comment (okonet) [reply src/index.js]: They will go away in #468

File: package.json (hunk 1 above)
Comment (okonet) [reply package.json]: This is added by `install-peerdeps` so I don't want to lock it

File: .eslintrc (hunk 4 above)
Comment (TrySound) [reply .eslintrc]: But source code is also esm. What's the problem?

File: package.json (hunk 2 above)
Comment (okonet) [reply package.json]: Same here

**Pull request merged**